build.gradle

```diff
@@ -216,6 +227,8 @@ task createBinDistZip(type: Zip) {
 
   appendix = 'bin'
   with distContent
+
+  dependsOn 'buildDTrace'
```

Comment:
To enable incremental building I would use `inputs.files buildDTrace.outputs` instead of `dependsOn 'buildDTrace'`.